DEPS

```diff
@@ -4,7 +4,7 @@ vars = {
   'libchromiumcontent_revision':
     'd6411e421b0b2f0105d274ac4e5c90dc979463ca',
   'node_version':
-    'ed9e26b86435bed3cf5e4dafa59aa90e85f7a4ab',
+    'f9b5b084480116ac512e96b79b258405d8501cf1',
```

Comment:
Was this included in the PR intentionally? This seems like a different topic

Reply:
@ckerr 
The change is intentional.
But it's in a separate commit with a "FIXME" prefix, the PR itself has the "[wip]" prefix.
It's only here to check how build times will be changed, without it Windows build fail on ninja files generation.